exchanges/exchange.go

```diff
@@ -1943,3 +1943,35 @@ func (b *Base) Bootstrap(_ context.Context) (continueBootstrap bool, err error)
 func (b *Base) IsVerbose() bool {
 	return b.Verbose
 }
+
+// GetDefaultConfig returns a default exchange config
+func GetDefaultConfig(ctx context.Context, exch IBotExchange) (*config.Exchange, error) {
+	if exch == nil {
+		return nil, errExchangeIsNil
+	}
+
+	if exch.GetName() == "" {
+		exch.SetDefaults()
+	}
+
+	b := exch.GetBase()
+
+	exchCfg, err := b.GetStandardConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	err = b.SetupDefaults(exchCfg)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This function is so weird! "get me the default config" means setup some parts of the exchange, reset some exchange values to the default config's and then make REST calls.

I'm not going to make you refactor this, it's how its been and refactoring isn't really going to improve much right now. Just a fun function